layouts/default.vue

```diff
@@ -26,6 +26,10 @@
   "ja": {
     "siteTitleL1": "原神 英語・",
     "siteTitleL2": "中国語辞典"
+  },
+  "zh-CN": {
+    "siteTitleL1": "Genshin",
+    "siteTitleL2": "Dictionary"
```

Comment:
I split the site title into two <span>s to avoid an unintended line break.

Reply:
Sorry, `<span>` was hidden in the above comment because I didn't enclose it with backquotes.